prow/cmd/build/main.go

```diff
@@ -190,7 +196,17 @@ func main() {
 		go runServer(o.cert, o.privateKey)
 	}
 
-	controller := newController(kc, pjc, pjif.Prow().V1().ProwJobs(), buildConfigs, o.totURL, configAgent.Config, kube.RateLimiter(controllerName))
+	// TODO remove the feature flag and use directly the config value
+	allowCancellations := false
+	if o.useAllowCancellations {
+		cfg := configAgent.Config()
+		if cfg != nil {
+			allowCancellations = configAgent.Config().Plank.AllowCancellations
```

Comment:
config changes at runtime, please refactor this to load the live value whenever it is called like plank does

Reply:
I refactored the code to load dynamically the value from config. The value is gated by the feature flag `useAllowCancellations`.